security/proto/providers/google/meshca.proto

```diff
@@ -14,16 +14,22 @@
 
 syntax = "proto3";
 
-package google.security.meshca.v1beta1;
+package google.security.meshca.v1;
+
+import "google/protobuf/duration.proto";
 
 // Certificate request message.
 message MeshCertificateRequest {
+  // The request ID must be a valid UUID with the exception that zero UUID is
+  // not supported (00000000-0000-0000-0000-000000000000).
+  string request_id = 1;
```

Comment:
Seems a bit weird to change the proto #, isn't it supposed to be immutable

Reply:
Yes, but this is an API change (different package), so it's OK not to be backward compatible :)